eng/ci/code-mirror.yml

```diff
@@ -0,0 +1,21 @@
+trigger:
+  branches:
+    include:
+    # Keep this set limited as appropriate (don't mirror individual user branches).
+    - main
+    - release/*
+    - v3.x
+    - v4.x
```

Comment:
Not sure which branches to keep here out of below set -

`main`, `v3.x`, `v4.x`, `release/1`, `release/4`, `release/v1.0`

Reply:
Include whichever branches are actively worked on and released from in this repository.